### DISCUSSION TITLE
Camera Gestures

### DISCUSSION THREAD

**Discussion by ghost** [sceneview/sceneview#570] [Q&A]
I just want to disable all camera gestures. I've looked around but couldn't find anything. Help please.

**Comment by ThomasGorisse** [comment]
In v4.0.0-rc.1, camera gestures are configured via `cameraManipulator`:

```kotlin
SceneView(
    cameraManipulator = rememberCameraManipulator(
        orbitHomePosition = Position(0f, 0f, 4f),
        targetPosition = Position(0f, 0f, 0f)
    )
) {
    ModelNode(modelInstance = model)
}
```

By default, `SceneView` supports orbit (single finger drag), pan (two finger drag), and zoom (pinch). You can customize or disable individual gestures through the camera manipulator parameters.
